plugin/src/steps/check-plugin-requirements.ts

```diff
@@ -53,6 +58,7 @@ const areRemotePluginVersionsSatisfied = async ({
 
   try {
     const { data } = await fetchGraphql({
+      url: wpGraphQLEndpoint,
```

Comment:
Do we need to add this? I thought `fetchGraphql` can access the url internally. I think we only need to pass the url explicitly for our tests or anywhere that we're making a request to an API besides WPGraphQL.

Reply:
ah! ok so we need to make `url` optional then? this is one other runtime change i forgot about!